addons/mail/static/src/components/chat_window_header/chat_window_header.js

```diff
@@ -37,6 +37,14 @@ class ChatWindowHeader extends Component {
         return this.env.models['mail.chat_window'].get(this.props.chatWindowLocalId);
     }
 
+    /**
+     * @returns {string}
+     */
+    get channelType() {
+        const chatWindow = this.chatWindow;
+        return chatWindow.thread && chatWindow.thread.channel_type;
+    }
```

Comment:
Remove this one and access values directly in the template. Only "top level" records used in the component should have getters.